layouts/_default/split.html

```diff
@@ -6,27 +6,24 @@
   <body class="{{ printf "bg-%s" $bg }}">
     {{ partial "nav.html" . }}
 
-    {{ $page := replace (strings.TrimRight "/" (strings.TrimLeft "/" .RelPermalink)) "/" "__" }}
-
-    {{ with index $.Site.Data $page }}
-      {{ range . }}
-        {{ $.Scratch.Set "data" . }}
-
-        {{ if and (not .disabled) (eq .fragment "hero") }}
-          {{ partial "fragments/hero.html" $ }}
-        {{ end }}
+    {{ range sort .Resources }}
 
+      {{ if and (not .Params.disabled) (eq .Params.fragment "hero") }}
+        {{ partial "fragments/hero.html" .Params }}
       {{ end }}
+
     {{ end }}
 
+    {{ $page := replace (strings.TrimRight "/" (strings.TrimLeft "/" .RelPermalink)) "/" "__" }}
+
     {{ partial "content-split.html" . }}
 
     {{ with index $.Site.Data $page }}
       {{ range sort . "weight" }}
         {{ $.Scratch.Set "data" . }}
-
-        {{ if and (not .disabled) (eq .fragment "items") }}
-          {{ partial "fragments/items.html" $ }}
+        
+        {{ if and (not .Params.disabled) (eq .Params.fragment "items") }}
+          {{ partial "fragments/items.html" .Params }}
```

Comment:
Same thing as above.

Reply:
{{ . Resource }}  on line 21 is supposed to be on line 25. I used {{ range sort .Resource }} initially only get an error that split.html is an incomplete or empty template.